src/qt/walletmodel.cpp

```diff
@@ -733,6 +733,11 @@ bool WalletModel::hdEnabled() const
     return wallet->IsHDEnabled();
 }
 
+OutputType WalletModel::getDefaultAddressType() const
+{
+    return g_address_type;
```

Comment:
Nit, `assert(g_address_type != OUTPUT_TYPE_NONE)` ping @sipa.

Reply:
This feature doesn't really care if OUTPUT_TYPE_NONE Is returned though; the checkbox would be disabled. So is this really the place for an assert? If so, I'm happy to add it.

Reply:
I think this is fine. NONE will be caught if it's passed down and actually used somewhere.